### DISCUSSION TITLE
Welcome to tiro-notes Discussions!

### DISCUSSION THREAD

**Discussion by dotgreg** [dotgreg/tiro-notes#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by freecicero** [comment]
This application looks great so far and I hope you will continue your work.  Just wanted to register a word of encouragement!  I'd like to use this in place of Qownnotes to work with a Notes folder that I am keeping in sync with Nextcloud, and so far it seems to be working great.  Thank you!
